recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "1.3.1" %}
+{% set version = "1.3.2" %}
 
 package:
   name: kiwisolver
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/k/kiwisolver/kiwisolver-{{ version }}.tar.gz
-  sha256: 950a199911a8d94683a6b10321f9345d5a3a8433ec58b217ace979e18f16e248
+  sha256: fc4453705b81d03568d5b808ad8f09c77c47534f6ac2e72e733f9ca4714aa75c
 
 build:
   number: 0
```

Comment:
Maybe it would be better to use `skip: True. # [py<37]`

Reply:
The `skip` section has beed added to the recipe

https://github.com/AnacondaRecipes/kiwisolver-feedstock/blob/update_to_1.3.2/recipe/meta.yaml#L12